src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SearchBar from '@theme-original/SearchBar';
+
+export default function SearchBarWrapper(props) {
+  return (
+    <>
```

Comment:
This fragment isn't necessary.

Reply:
Removing this section (and the file, too) breaks the build. Can you advise?

Reply:
Hi @irontitan76 , to clarify, were you advising that we remove the whole SearchBar component (src/theme/SearchBar/index.js)? I've tried doing this, but without it, the Algolia search bar won't appear in the nav.